### PR TITLE
[ASI] Image details, editing, search and metadata

### PR DESCRIPTION
**Pull request by lenaorobei**
## Purpose of this pull request

This pull request (PR):
- fixes https://github.com/magento/adobe-stock-integration/issues/1535
- fixes https://github.com/magento/adobe-stock-integration/issues/1534
- fixes https://github.com/magento/adobe-stock-integration/issues/1533 .

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs-beta.magento.com/user-guide/cms/media-gallery-asset-management.html

whatsnew
[Media Gallery Asset Management](https://docs.magento.com/user-guide/cms/media-gallery-asset-management.html) was updated to include expanded asset details, as well as new asset search and image metadata editing.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cms/media-gallery-asset-management.md [hunk 1]
```diff
@@ -23,11 +23,42 @@ The new [Media Gallery]({% link cms/media-gallery.md %}) provides tools for mana
...
+
+   - **Categories**
+   - **Products**
+   - **Pages**
+   - **Blocks**
 
     ![]({% link images/images/enhanced-media-gallery/asset-details.png %}){: .zoom}
     _Asset Details_
 
+    Click **Used In** links to see the details. The grid in the following example shows all categories where specific asset is used.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:36]:
```suggestion
    Click the **Used In** links to see the details. The grid in the following example shows all categories where a specific asset is used.
```

File: src/cms/media-gallery-asset-management.md [hunk 2]
```diff
@@ -23,11 +23,42 @@ The new [Media Gallery]({% link cms/media-gallery.md %}) provides tools for mana
...
+
+    ![]({% link images/images/enhanced-media-gallery/asset-category-grid.png %}){: .zoom}
+    _Category Grid_
+
+    It is also possible to delete the asset from the **View Details** section.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:41]:
```suggestion
    It is also possible to delete the asset from the _View Details_ section.
```

File: src/cms/media-gallery-asset-management.md [hunk 3]
```diff
@@ -23,11 +23,42 @@ The new [Media Gallery]({% link cms/media-gallery.md %}) provides tools for mana
...
+## Edit an asset
+
+1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
+
+1. Click the three dots below the asset (![]({% link images/images/enhanced-media-gallery/three-dots-btn.png %})), then click <span class="btn">Edit</span>.
+
+    ![]({% link images/images/enhanced-media-gallery/edit-asset.png %}){: .zoom}
+    _Edit Asset_
+
+1. You can change following fields though **Edit Image** form.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:52]:
```suggestion
1. If needed, change and of the following metadata values:
```

File: src/cms/media-gallery-asset-management.md [hunk 4]
```diff
@@ -23,11 +23,42 @@ The new [Media Gallery]({% link cms/media-gallery.md %}) provides tools for mana
...
+
+   - **Title**
+   - **Description**
+   - **Tags/Keywords**
+
+   Please note, that this data will be saved to the data base as well as to the file metadata itself. Currently, **XMP** and **IPTC* formats are supported.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:58]:
```suggestion
   This data will be saved to the data base as well as to the file metadata itself. Currently, XMP and IPTC formats are supported.
```

File: src/cms/media-gallery-asset-management.md [hunk 5]
```diff
@@ -53,6 +84,19 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
     ![]({% link images/images/enhanced-media-gallery/bulk-delete-asset-confirmation.png %}){: .zoom}
     _Delete Confirmation_
 
+## Search for assets
+
+1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
+
+1. Use **Search by keywords** input to perform image search by keywords/tags.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:91]:
```suggestion
1. Use the **Search by keywords** input to perform image search by keywords/tags.
```

File: src/cms/media-gallery-asset-management.md [hunk 6]
```diff
@@ -53,6 +84,19 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
...
+
+    The search in the following example finds assets that contain a specific tag (`mountain`).
+
+    ![]({% link images/images/enhanced-media-gallery/asset-search.png %}){: .zoom}
+    _Asset Search_
+
+1. See [Edit an asset][edit-asset] section to learn how you can update image tags.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:98]: This really isn't a step

```suggestion
{:.bs-callout-info}
See the [Edit an asset][edit-asset] section to learn how you can update image tags.
```

**Comment by lenaorobei** [comment]
@leslietilling thank you for the review.
Suggestions were applied.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**